README.md

```diff
@@ -13,7 +13,7 @@ The `@lukso/eip191-signer.js` package is used to sign an EIP191 Execute Relay Ca
 
 This library will add the `\x19Execute Relay Call:\n` prefix to a message and sign it.
 
-The `EIP191 ExcuteRelayCall` prefix is used instead of the standard Ethereum transaction prefix to sign messages so that an [`executeRelayCall`](https://docs.lukso.tech/standards/smart-contracts/eip191-key-manager#executerelaycall) transaction cannot be inadvertently signed when signing an Ethereum signed message.
+The `EIP191 ExcuteRelayCall` prefix is used instead of the standard Ethereum transaction prefix to sign messages so that an [`executeRelayCall`](https://docs.lukso.tech/standards/smart-contracts/lsp6-key-manager#executerelaycall) transaction cannot be inadvertently signed when signing an Ethereum signed message.
```

Comment:
```suggestion
`EIP191` is used instead of the standard Ethereum transaction prefix to differentiate between signing generic messages _vs_ signing transactions that aim to be executed via [`executeRelayCall`](https://docs.lukso.tech/standards/smart-contracts/lsp6-key-manager#executerelaycall). This distinction enables to prevent from signing inadvertently data that correspond to relayed transactions for execution via the [LSP6KeyManager](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-6-KeyManager.md).
```